--- /tmp/llvm/lib/cmake/llvm/LLVMExports.cmake	2024-09-13 18:42:39.000000000 +0000
+++ /tmp/llvm/lib/cmake/llvm/LLVMExports.cmake.b	2024-09-17 18:32:33.018437267 +0000
@@ -1700,7 +1700,7 @@
 foreach(_cmake_target IN LISTS _cmake_import_check_targets)
   foreach(_cmake_file IN LISTS "_cmake_import_check_files_for_${_cmake_target}")
     if(NOT EXISTS "${_cmake_file}")
-      message(FATAL_ERROR "The imported target \"${_cmake_target}\" references the file
+      message(DEBUG "The imported target \"${_cmake_target}\" references the file
    \"${_cmake_file}\"
 but this file does not exist.  Possible reasons include:
 * The file was deleted, renamed, or moved to another location.
